src/bilder/components/vector/templates/global_metric_sink.yaml

```diff
@@ -0,0 +1,13 @@
+---
+sinks:
+  ship_host_metrics_to_grafana_cloud:
+    inputs:
+    # Expects the final step of ALL metric pipelines to be named 'noop_metric_global_funnel'
+    - 'noop_metric_global_funnel'
+    type: prometheus_remote_write
+    endpoint: https://prometheus-prod-10-prod-us-central-0.grafana.net/api/prom/push
+    healthcheck: false
+    auth:
+      strategy: basic
+      user: "${GRAFANA_CLOUD_PROMETHEUS_API_USER}"
+      password: ${GRAFANA_CLOUD_API_KEY}
```

Comment:
I think it might be better to convert EVERYTHING to use these shared sinks before converting them to consul/vault templates. There is a validate step during install+config that will break with switching to templates and having both co-existing will be complicated, I think.

Reply:
What I mean is to use the vault/consul templates to write the values into `/etc/default/vector` and leave the global sink yaml files as-is

Reply:
I've been poking at this for a bit now and it keeps getting larger and larger in scope and consequences so I think I'm going to keep it for another day. We will need an `infrastructure` stack to store the grafana-cloud secrets into vault. Getting the `datacenter` is not entirely trivial, which is going to be used for 'environment' in the final log / metric document shipped off to Grafana. There are a few other things as well.